### PR TITLE
{tools}[GCCcore/11.3.0] Subversion v1.14.2

### PR DESCRIPTION
**Pull request by SebastianAchilles**
(created using `eb --new-pr`)

Depends on:
- `APR`, `APR-util` https://github.com/easybuilders/easybuild-easyconfigs/pull/16687
- `Serf` https://github.com/easybuilders/easybuild-easyconfigs/pull/16688

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2

**Review by branfosj — DISMISSED** [review]

File: easybuild/easyconfigs/s/Subversion/Subversion-1.14.2-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    ('Autotools', '20220317'),
+]
+
+dependencies = [
+    ('APR', '1.7.0'),
+    ('APR-util', '1.6.1'),
+    ('SQLite', '3.38.3'),
+    ('zlib', '1.2.12'),
+    ('lz4', '1.9.3'),
+    ('utf8proc', '2.8.0'),
```
Comment (branfosj) [inline easybuild/easyconfigs/s/Subversion/Subversion-1.14.2-GCCcore-11.3.0.eb:35]:
```suggestion
    ('utf8proc', '2.7.0'),
```

We have https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/u/utf8proc/utf8proc-2.7.0-GCCcore-11.3.0.eb

Or is there a need to use a newer version?

**Review by branfosj — CHANGES_REQUESTED** [review]

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
bear-pg0104u04b - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/3fda11edd3815720c430c9e473524ebe for a full test report.

File: easybuild/easyconfigs/s/Subversion/Subversion-1.14.2-GCCcore-11.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+dependencies = [
+    ('APR', '1.7.0'),
+    ('APR-util', '1.6.1'),
+    ('SQLite', '3.38.3'),
+    ('zlib', '1.2.12'),
+    ('lz4', '1.9.3'),
+    ('utf8proc', '2.7.0'),
+    ('Serf', '1.3.9'),
+]
```
Comment (branfosj) [inline easybuild/easyconfigs/s/Subversion/Subversion-1.14.2-GCCcore-11.3.0.eb:37]:
```suggestion
    ('util-linux', '2.38'),
]
```

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0104u04b - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/a95a8fd2c4413dcbd72cce904455c8cb for a full test report.

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Going in, thanks @SebastianAchilles!

**Pull request merged**